pkg/generated/openapi/cmd/models-schema/main.go

```diff
@@ -59,7 +59,7 @@ func output() error {
 	if err != nil {
 		return fmt.Errorf("error serializing api definitions: %w", err)
 	}
-	_, err = os.Stdout.Write(data)
+	_, _ = os.Stdout.Write(data)
```

Comment:
We don't need to catch the result at all, then.  `os.Stdout.Write(data)` alone would suffice?

Reply:
good point